onnx/defs/tensor/defs.cc

```diff
@@ -1657,4 +1567,119 @@ ONNX_OPERATOR_SET_SCHEMA(
           updateOutputElemType(ctx, 0, TensorProto::INT64);
         }));
 
+static const char* Pad_ver10_doc = R"DOC(
+Given `data` tensor, pads, mode, and value.
+Example:
+  Insert 0 pads to the beginning of the second dimension.
+  data = [
+      [1.0, 1.2],
+      [2.3, 3.4],
+      [4.5, 5.7],
+  ]
+  pads = [0, 2, 0, 0]
+  output = [
+      [
+          [0.0, 0.0, 1.0, 1.2],
+          [0.0, 0.0, 2.3, 3.4],
+          [0.0, 0.0, 4.5, 5.7],
+      ],
+  ]
+)DOC";
+
+ONNX_OPERATOR_SET_SCHEMA(
+    Pad,
+    10,
+    OpSchema()
+        .Attr(
+            "mode",
+            "Three modes: constant(default), reflect, edge",
```

Comment:
Add more description or reference to clarify these modes?

Reply:
Updated.